external/netstandard/netstandard.depproj

```diff
@@ -49,6 +49,9 @@
     <PackageReference Include="System.Runtime.WindowsRuntime">
       <Version>4.3.0</Version>
     </PackageReference>
+    <PackageReference Condition="'$(NETStandardVersion)' &gt;= 1.1" Include="System.Runtime.WindowsRuntime.UI.Xaml">
```

Comment:
this should go into harvestPackages: https://github.com/dotnet/corefx/blob/master/external/harvestPackages/harvestPackages.props

Reply:
harvestPackages.props already contians this package https://github.com/dotnet/corefx/blob/master/external/harvestPackages/harvestPackages.props#L97

Reply:
This is correct, this makes the reference available for netstandard projects.